homeassistant/components/epic_games_store/helper.py

```diff
@@ -0,0 +1,101 @@
+"""Helper for Epic Games Store."""
+import contextlib
+from typing import Any
+
+from homeassistant.util import dt as dt_util
+
+
+def get_country_from_language(language: str) -> str:
+    """Get the country code from language."""
+    excepts = {"ja": "JP", "ko": "KR", "zh-Hant": "CN"}
```

Comment:
I don't think `"zh-Hant": "CN"` is correct, traditional Chinese writing is not used in mainland China. It is however used in Hong Kong, Singapore and Shanghai.

I'd suggest to double check where the epic game store is available, and add options for that to the config flow instead of trying to guess it here.

Reply:
Fixed with 7a53234, the only thing the country change is the currency & price.

Tested with weird combinaisons, and working fine.
I put the selector is the config flow so Chinese people living in France (or anywhere else 😄) could get the local currency & price, with his native language 😉